src/winforms/toga_winforms/widgets/splitcontainer.py

```diff
@@ -0,0 +1,42 @@
+from toga_winforms.libs import WinForms
+from toga_winforms.window import WinFormsViewport
+
+from .base import Widget
+
+
+class SplitContainer(Widget):
+
+    def create(self):
+        self.native = WinForms.SplitContainer()
+        if self.interface.direction == self.interface.HORIZONTAL:
```

Comment:
It looks to me that the interface constructor will set the initial direction, so it is not necessary to set it in `create()`

*toga_core/widgets/splitcontainer.py*
```python
def __init__(...):
        #  ...
        self._impl = self.factory.SplitContainer(interface=self)
        
        self.direction = direction #  indirectly calls `self._impl.set_direction()`
```

Reply:
Oh, sorry, my mistake. I was looking at gtk implementation, and didn't realize that it checks orientation of `splitcontainer` because in gtk the actual widgets for horizontal and vertical splitcontainer are different, which is not the case in winforms.